0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -318,3 +187,4 @@ Content-Type: application/json
 }
 ```
 [Try this request](https://red.ilpdemo.org/.well-known/webfinger?resource=acct%3Abob%40red.ilpdemo.org)
```

Comment:
Why is the SPSP `rel` called `receiver` and not `spsp`? That made more sense when there were several endpoints.